lib/pages/task_list.dart

```diff
@@ -29,14 +29,17 @@ class _TaskListState extends State<TaskList> {
                 color: iconColor,
               ),
               title: Text(
-                taskName,
+                taskName.getName(),
                 style: tilesText,
               ),
               trailing: GestureDetector(
                   onTap: () {
                     context.read<TaskCubit>().completeTask(
-                        taskName: taskName); //add context.read to use cubit
-                    context.read<TaskCubit>().removeTask(taskName: taskName);
+                        taskName:
+                            taskName); //add context.read to use cubit
+                    context
+                        .read<TaskCubit>()
+                        .removeTask(taskName: taskName);
```

Comment:
I think this needs formatting?? Maybe? Either way, it's not a big deal because you are working ahead here and the card isn't meant to deal with this yet and you aren't breaking anything. Just please make sure to format in general (it seems like you usually do). Please resolve this comment.

Reply:
Hm, it doesn't look that way for me. Weird
https://gyazo.com/d349235f4937a00f5f7af92a1fa10d05
![image](https://github.com/skeptickal/TaskListApp/assets/133704012/97bba0a6-84d2-4919-b296-6314f8de8394)